### ISSUE TITLE
Template engine

### ISSUE THREAD

**Issue by egoist** [egoist/deravel#3]
This one should be trivial, I think we can reuse `nunjucks` from Node.js?

**Comment by IniZio** [comment]
So will need to be able to install nodejs dependencies in deno or mirror the functions?

**Comment by egoist** [comment]
@IniZio template engine usually doesn't rely on Nodejs-specific modules that much, we can bundle one (like nunjucks) into a single file and convert it to es module.

**Comment by ecreeth** [comment]
Deno support JSX. What better than that 😄

**Comment by egoist** [comment]
@ecreeth yeah almost forgot that 😂 though I think the performance of a template engine would be slightly better.
